apps/artemis-web/libs/passport-strategies/local-strategy.ts

```diff
@@ -24,6 +25,7 @@ export const LocalStrategy = new Strategy(
         );
```

Comment:
hmm here since the user is authenticated (local strategy), shouldn't also the counter be reset?

Reply:
I added this reset to the /credentials.ts and /ldap.ts files